server/model/taggedBailSelectCaseModel.ts

```diff
@@ -47,6 +52,7 @@ export default class TaggedBailSelectCaseModel {
         ...it.sentences.sort((a, b) => new Date(a.sentenceDate).getTime() - new Date(b.sentenceDate).getTime())[0],
         selected: relevantSentenceForTaggedBailAdjustment(it, this.adjustment),
         selectCaseHref,
+        recall: isSentenceRecalled(it.sentences[0].sentenceCalculationType),
```

Comment:
I think this logic might need to be checked. Is it the first sentence on the court case that defines if its recall? What if the court case has a mix of some sentences recalled and some not? Can that happen? (Might be worth talking to Steve and potentially raising a separate ticket.